eng/Version.Details.xml

```diff
@@ -28,7 +28,7 @@
       <Sha>3f43bf1b2dead2cb51f20dc47f6dfd7981248820</Sha>
       <SourceBuild RepoName="sourcelink" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="Microsoft.DiaSymReader" Version="2.0.0" CoherentParentDependency="Microsoft.Net.Compilers.Toolset">
```

Comment:
What does this attribute even mean????

Reply:
It means that DiaSymReader's version should be taken from the version that roslyn depends on at the sha from Microsoft.NET.Compilers.Toolset:

https://github.com/dotnet/roslyn/pull/68523/files#diff-fb62e94a1d6f29f863e3d0a22aa38269f6cd1d7f03b109dc06e2cbf2548b86d3R35-R38

That creates a nice circular reference you can't update, and besides that, roslyn doesn't depend on DiaSymReader there.